router/event.go

```diff
@@ -124,3 +124,23 @@ func transformHeaders(req http.Header) map[string]string {
 
 	return headers
 }
+
+
+func correctEventData(event *eventpkg.Event, body []byte, mime string) error {
+	if len(body) > 0 {
+		switch mime {
+		case mimeJSON:
+			err := json.Unmarshal(body, &event.Data)
+			if err != nil {
+				return errors.New("malformed JSON body")
+			}
+			break
+		default:
+			if byteSlice, ok := event.Data.([]byte);
+				ok && (strings.HasPrefix(mime, mimeFormMultipart) || mime == mimeFormURLEncoded) {
```

Comment:
Why you check for prefix here, not the full header?

Reply:
Because the content type is different based on the user agent. For example, Insomnia send this as content type: `multipart/form-data; boundary=X-INSOMNIA-BOUNDARY`.